### ISSUE TITLE
Keyboard navigation doesn't work if there is no selected date

### ISSUE THREAD

**Issue by sstoyanovIG** [IgniteUI/igniteui-angular#2828]
### Description  
Keyboard navigation doesn't work if there is no selected date, because the focus is on the dialog itself.

 * igniteui-angular version: 6.2.x
 * browser: all

### Steps to reproduce  

1. Open  https://staging.infragistics.local/products/ignite-ui-angular/angular/components/date_picker.html#usage
2. Navigate to "Default Date Picker" sample
3. Click date-picker

### Result  
Keyboard navigation doesn't work.

### Expected result  
Keyboard navigation should work.
